### PR TITLE
[stable/jaeger-operator] Add service definition

### PR DESCRIPTION
**Pull request by mwieczorek**
#### What this PR does / why we need it:
This PR adds a service definition for jaeger-operator. It's required for configuring `servicemonitor` when using `prometheus-operator`.
Jaeger-operator creates a service automatically but with different labels so no endpoints are 'attached' (that's why we need a custom one). 
I added the suffix `-metrics` for service name to avoid a conflict with the service name created by the operator.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:
Chart version will be in conflict with https://github.com/helm/charts/pull/13964 but I'll rebase if one of it will be merged.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by mwieczorek** [comment]
@davidkarlsen - will you be able to take a look on that?
if not, maybe @cpanato? (as you reviewed my last PR: https://github.com/helm/charts/pull/13964)

File: stable/jaeger-operator/templates/service.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "jaeger-operator.fullname" . }}-metrics
+  labels:
+    app.kubernetes.io/name: {{ include "jaeger-operator.name" . }}
```
Comment (cpanato) [inline stable/jaeger-operator/templates/service.yaml]: use the {{ include "jaeger-operator.labels" . | indent 4 }}

**Review by cpanato — CHANGES_REQUESTED** [review]

File: stable/jaeger-operator/templates/service.yaml (hunk 1 above)
Comment (mwieczorek) [reply stable/jaeger-operator/templates/service.yaml]: changed

**Comment by mwieczorek** [comment]
@cpanato @batazor - could you take a look?

**Comment by mwieczorek** [comment]
@cpanato Could you take a look? Thanks.

**Review by cpanato — APPROVED** [review]

**Pull request merged**